plugins/mcreator-localization/lang/texts.properties

```diff
@@ -73,6 +73,8 @@ action.gradle.close_mcreator_while_running_title=Close workspace
 dialog.ai_condition.additional_start=Additional start condition
 dialog.ai_condition.additional_continue=Additional continue condition
 dialog.ai_condition.panel_name=AI condition editor
+dialog.arg_procedure.panel_name=Parameter procedure editor
+dialog.arg_procedure.execute_action=Execute action procedure
```

Comment:
Where is this key used? Please re-check keys, you did not check them fully yesterday I asked